test/BondMinter.ts

```diff
@@ -104,8 +104,14 @@ describe("BondConfigVault", () => {
 
       await time.setNextBlockTimestamp(timestampAfter);
 
-      await bondMinter.mintBonds();
-
+      expect(await bondMinter.mintBonds())
```

Comment:
await has to be outside the expect to assert events

Reply:
ah. I was curious why the linter was complaining